.github/workflows/python_ci.yml

```diff
@@ -29,7 +29,9 @@ jobs:
         if: matrix.os == 'windows-latest'
         run: |
           choco install -y graphviz
-
+          New-ItemProperty -Path "HKLM:\SYSTEM\CurrentControlSet\Control\FileSystem" -Name "LongPathsEnabled" -Value 1 -PropertyType DWORD -Force
```

Comment:
I did see random failures on Windows if it was due to long paths then thanks so much for fixing that.